spec/models/contact_form_spec.rb

```diff
@@ -0,0 +1,95 @@
+
+
+#  `rails generate curation_concerns:work GenericWork`
+
+require 'rails_helper'
+
+describe ContactForm do
+
+  describe "it requires input" do
+
+    before do
+
+      subject.contact_method = "Contact Menthod"
+      subject.category = "Category"
+      subject.name = "John Smith"
+      subject.email = "abc@123.com"
+      subject.subject = "This is the subject"
+      subject.message = "This is a message."
+
+    end
+
+
+    it "validates contact_method" do
+
+      subject.contact_method = nil
+      expect(subject).to be_valid
+
+    end
+
+
+
+    it "validates category" do
+
+      expect(subject).to be_valid
+      subject.category = nil
+      expect(subject).not_to be_valid
+
+    end
+
+
+
+    it "validates name" do
+
+      expect(subject).to be_valid 
+      subject.name = nil
+      expect(subject).not_to be_valid
+
+    end
+
+
+
+    it "validates email" do
+
+      expect(subject).to be_valid 
+      subject.email = nil
+      expect(subject).not_to be_valid
+
+    end
+
+
+
+    it "validates subject" do
+
+      expect(subject).to be_valid 
+      subject.subject = nil
+      expect(subject).not_to be_valid
+
+    end
+
+    it "validates message" do
+
+      expect(subject).to be_valid 
+      subject.message = nil
+      expect(subject).not_to be_valid
+
+    end
+
+    it "validates header" do
```

Comment:
This seems like a spec regarding the header function filling in the proper attributes.  Validation probably isn't the right description for this test.